apps/src/p5lab/AnimationPicker/AnimationPicker.jsx

```diff
@@ -52,13 +53,15 @@ class AnimationPicker extends React.Component {
     uploadInProgress: PropTypes.bool.isRequired,
     uploadError: PropTypes.string,
     is13Plus: PropTypes.bool,
+    selectedAnimations: PropTypes.arrayOf(PropTypes.object).isRequired,
     onClose: PropTypes.func.isRequired,
     onPickNewAnimation: PropTypes.func.isRequired,
     onPickLibraryAnimation: PropTypes.func.isRequired,
     onUploadStart: PropTypes.func.isRequired,
     onUploadDone: PropTypes.func.isRequired,
     onUploadError: PropTypes.func.isRequired,
-    playAnimations: PropTypes.bool.isRequired
+    playAnimations: PropTypes.bool.isRequired,
+    onAnimationSelectionDone: PropTypes.func.isRequired
```

Comment:
naming nit: i think i usually see us refer to a callback like this as `onComplete` (e.g., `onAnimationSelectionComplete`)